hbt/calibration/default.py

```diff
@@ -46,12 +55,25 @@ def default(self: Calibrator, events: ak.Array, **kwargs) -> ak.Array:
     # !! so no manual sorting needed here (but necessary if, e.g., jec is applied before)
     events = self[custom_deterministic_event_seeds](events, **kwargs)
     events = self[deterministic_jet_seeds](events, **kwargs)
+    events = self[deterministic_electron_seeds](events, **kwargs)
 
+    events = self[electron_sceta](events, **kwargs)
```

Comment:
Can we add a switch here that runs the producer if the field is not existing?
(or is it never existing in the first place?)